### PR TITLE
CodeceptJS: Updated search index

### PR DESCRIPTION
**Pull request by DavertMik**
We have updated our site, indexes should be updated too

# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### CODE REVIEW

File: configs/codecept.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "//*[contains(@class,'navGroups')]//*[contains(@class,'navListItemActive')]/preceding::h3[1]",
-      "type": "xpath",
+      "selector": ".article-title",
+      "type": "css",
```
Comment (s-pace) [inline configs/codecept.json]:
```suggestion
```

**Comment by DavertMik** [comment]
Done!

**Comment by DavertMik** [comment]
How is the index updated? Does it require some time after this PR to rerun a crawler, or not?
I didn't test it on algolia locally, but all selectors are valid

**Comment by s-pace** [comment]
Deployed and fixed

**Pull request merged**